src/main/python/TravelTimeReporter.py

```diff
@@ -410,15 +410,23 @@ def coalesce_results(self):
         ).reset_index().fillna(self.settings["infinity"]).sort_values(
             ["i", "j"]
         )
+
+        _ebikeMaxTime = self.constants["ebikeMaxDist"] / self.constants["ebikeSpeed"] * 60
+        _escooterMaxTime = self.constants["escooterMaxDist"] / self.constants["escooterSpeed"] * 60
+
+        _ebikeTime = self.results["bike"] * self.constants["bikeSpeed"] / self.constants["ebikeSpeed"] + self.results["i"].map(self.land_use["MicroAccessTime"]) + self.constants["microConstant"] + self.constants["microRentTime"]
```

Comment:
I believe microConstant does not represent actual additional time traveled (otherwise we would be adding an hour to all micromobility trips!)

Reply:
It's in the [mode choice spec files](https://github.com/SANDAG/ABM/blob/ABM3_develop/src/asim/configs/resident/trip_mode_choice.csv#L495), so if we were to take it out of the travel time calculator, we should probably remove it from those as well.

Reply:
I would like to remove microConstant from mode choice spec files eventually. However, if we remove it now, it might trigger a recalibration exercise. This is currently acting like a bias/alternative-specific constant and does not need to be included in travel time calculations. A similar example would be that we weight the transit access time more than the in-vehicle access time but do not apply that for calculating the travel time by transit. It is just perceived time.